content/en/docs/tasks/manage-kubernetes-objects/declarative-config.md

```diff
@@ -237,7 +237,7 @@ kubectl scale deployment/nginx-deployment --replicas=2
 Print the live configuration using `kubectl get`:
 
 ```shell
-kubectl get -f https://k8s.io/examples/application/simple_deployment.yaml -o yaml
+kubectl get deploy nginx-deployment -o yaml
```

Comment:
How about changing `deploy` to `deployment`, to make it clear what the command is requesting?

Reply:
Done